frontend/.env.example

```
@@ -1 +1,4 @@
-VITE_API_URL=/api
\ No newline at end of file
+VITE_API_URL=/api
+VITE_CONTRACT_ID=
+VITE_RPC_URL=https://soroban-testnet.stellar.org:443
+VITE_NETWORK_PASSPHRASE=Test SDF Network ; September 2015
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)